packages/peregrine/lib/talons/SearchPage/useSearchPage.js

```diff
@@ -2,29 +2,32 @@ import { useCallback, useEffect, useMemo, useRef } from 'react';
 import { useLazyQuery, useQuery } from '@apollo/client';
 import { useLocation } from 'react-router-dom';
 
-import { useAppContext } from '@magento/peregrine/lib/context/app';
-import { usePagination, useSort } from '@magento/peregrine';
-
+import { useAppContext } from '../../context/app';
+import { usePagination } from '../../hooks/usePagination';
+import { useScrollTopOnChange } from '../../hooks/useScrollTopOnChange';
 import { getSearchParam } from '../../hooks/useSearchParam';
+import { useSort } from '../../hooks/useSort';
 import { getFiltersFromSearch, getFilterInput } from '../FilterModal/helpers';
-import { useScrollTopOnChange } from '../../hooks/useScrollTopOnChange';
+import DEFAULT_OPERATIONS from './searchPage.gql';
 
 /**
  * Return props necessary to render a SearchPage component.
  *
  * @param {Object} props
  * @param {String} props.query - graphql query used for executing search
  */
-export const useSearchPage = props => {
+export const useSearchPage = (props = {}) => {
     const {
-        queries: {
-            filterIntrospection,
-            getProductFiltersBySearch,
-            productSearch,
-            getPageSize
-        }
+        operations = DEFAULT_OPERATIONS,
+        queries: { getPageSize }
```

Comment:
Merge conflict, I'm guessing.

Reply:
Bingo 🐶  This came in from a recent merge, and since we've decided to split up the rest of component work, left this alone. In the next grooming I wanted to talk about [PWA-983](https://jira.corp.magento.com/browse/PWA-983) and [PWA-984](https://jira.corp.magento.com/browse/PWA-984), and how we want to break up/prioritize the rest.

Once this is merged too, we can start enforcing this prop structure and colocation going forward, so that migration scope doesn't continue to expand.